### ISSUE TITLE
The problem of partitioning the CityScapes dataset

### ISSUE THREAD

**Issue by bjh111** [voletiv/mcvd-pytorch#33]
36/5000
Dear author, how do you divide the training set and test set of the CityScapes dataset?
